### DISCUSSION TITLE
Reference to the datasource

### DISCUSSION THREAD

**Discussion by kushalpuri0817** [kagkarlsson/db-scheduler#529] [Q&A]
My application connects to two databases and I need to reference the db schedular to reference to the specific database while loading my springboot application. How can I achieve it? Is there any property that I can set in application.properties file to refer to the right datasource?

**Comment by kagkarlsson** [comment]
Not sure if the spring-boot starter supports that. You will have to investigate. 

You can always set it up manually using the `SchedulerBuilder` and feed it whatever DataSource you like
